webui/tests/selenium/webui-selenium-test.py

```diff
@@ -196,6 +196,13 @@ def setUp(self):
                 self.driver = webdriver.Firefox(capabilities=d, firefox_profile=fp)
             else:
                 raise RuntimeError('Browser {} not found.'.format(str(self.browser)))
+            #
+            # set explicit window size
+            #
+            # Used by Univention AppCenter test.
+            self.driver.set_window_size(1200, 800)
```

Comment:
Making the resolution the test runs with configurable via environment variable in the testrunner would be great.

Reply:
Implemented. Use BAREOS_WEBUI_BROWSER_RESOLUTION variable.